Ilya/NetflixRoulette/netflix-roulette/src/shared/movieList/MovieList.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import "./MovieList.scss";
+import MovieCard from "shared/movieCard/MovieCard"
+import { Link, RouteComponentProps } from "react-router-dom";
+import { HomePageConnectProps } from 'pages/home';
+import { IMovie } from 'shared/interfaces/IMovie';
+
+type IMovieListProps = HomePageConnectProps & RouteComponentProps;
+
+class MovieList extends React.Component<IMovieListProps> {
+
+    render() {
+        return (
+            <main>
```

Comment:
Not a really good place for **main** tag. **main** should wrap whole main content on the page(content between header and footer). Don't keep inside of specific component unless it's some kind of Layout or Container component that wraps all of the content.

Reply:
Done